### PR TITLE
Add Release notes on v2.0.0

### PR DESCRIPTION
**Pull request by rawalexe**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Release Notes v2.0.0 (Dec 16 2024)
+
+This the first release for Greengrass nucleus lite (GGLite) product line. GGLite
+v2 aims to be compatable with AWS's GreenGrass v2(GGV2) product line, however
+currently only subset of the features are supported with this release.
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md]:
```suggestion
currently only a subset of the features are supported with this release.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Installing from sources
+
+To install GGLite from source, please follow the guide from
+[SETUP.md](./docs/SETUP.md) and [TES.md](./docs/TES.md), once the setup
+environment is complete, please start refering to
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md:12]:
```suggestion
 is complete, please start refering to
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+[INSTALL.md](./docs/INSTALL.md) file.
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md:13]:
```suggestion
[INSTALL.md](./docs/INSTALL.md).
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## New on this release
+
+AWS takes pride in open source projects and support from community in building
+these projects. To provide more transparency on decisions and be in sync with
+the community all the design docs and specs for the project are now provided
+with this release. You can find more details on the [design doc](./docs/design/)
+and [spec doc](./docs/spec/) by following the corresponding links.
+
+As mentioned at the start, currently GGLite isn't fully compatable with GGV2.
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md]: Is GGV2 the name of the Java greengrass library?

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+The plan is to release this initail version of GGLite with basic feature support
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md]:
```suggestion
The plan is to release this initial version of GGLite with basic feature support
```

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+## New with this release
+
+To provide more transparency on decisions and be in sync with the community all
+the design docs and specs for the project are now provided with this release.
+You can find more details on the [design doc](./docs/design/) and
+[spec doc](./docs/spec/) by following the corresponding links.
+
+As mentioned at the start, currently nucleus lite isn't fully compatable with
+GGV2. The plan is to release this initail version of nucleus lite with basic
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md:32]: GGV2 - is that the official name of the product?

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+feature support first and then gain full feature compatibility with future
+releases. Currently only basic recipe types are supported and inforation on the
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md:34]:
```suggestion
releases. Currently only basic recipe types are supported and information on the
```

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+supported recipe can be found here [here](./docs/RECIPE_SUPPORT_CHANGES.md).
+
+Furthermore there are known difference in behavior that are outlined by
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md:37]:
```suggestion
Furthermore there are known differences in behavior that are outlined by
```

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+[KNOWN_ISSUES](./docs/KNOWN_ISSUES.md) document. Also please look at the Github
+issues section to keep up with latest known issues. It's encouraged to open up a
```
Comment (AniruddhaKanhere) [inline RELEASE_NOTES.md:39]:
```suggestion
issues section to keep up with latest known issues. Please create a
```

**Review by AniruddhaKanhere — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 2 above)
Comment (rawalexe) [reply RELEASE_NOTES.md:12]: I want to keep the setup environment as this is in relation with setting up a environment or the follow up where a developer can setup a nix environment

**Comment by AniruddhaKanhere** [comment]
@rawalexe can we use this text instead?

# Release Notes v2.0.0 (Dec 16 2024)

This the first release for Greengrass nucleus lite. It aims to be compatable with AWS's GreenGrass v2(GGV2) product line, however currently only a subset of the features are supported with this release.

## Installing from source

To install Greengrass nucleus lite from source, please follow the setup guide [SETUP.md](./docs/SETUP.md) and [TES.md](./docs/TES.md), once the development environment is setup, please refer to [INSTALL.md](./docs/INSTALL.md).

## Contribution guidelines

Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, code, or documentation, we welcome our community to be involved in this project.

It's recommended to setup a nix environment. Information for nix environment setup can be found [here](./docs/DEVELOPMENT.md).

For provisioning nucleus lite device, fleet provisioning by claim can be used. To setup fleet, please take a look at the fleet provisioning setup guide [here](./docs/Fleet-provisioning.md).

## New with this release

The desing document for the Greengrass nucleus lite can be found in the [design](./docs/design/) and [spec](./docs/spec/) directories.

Greengrass nucleus lite is not fully compatible with GGV2. Currently, Greengrass nucleus lite only has basic feature support. The follwoing releases will add nore feature to reach feature parity with GGV2. The basic recipe types that are presently supported along with detailed information can be found [here](./docs/RECIPE_SUPPORT_CHANGES.md).

## Known issues

The differences between Greengrass nuclues lite and GGV2 along with the currently known issues can be found in the [KNOWN_ISSUES](./docs/KNOWN_ISSUES.md) document.

We welcome you to create an issue to report bugs or suggest features.

**Review by thomas-roos — APPROVED** [review]

**Review by thomas-roos — APPROVED** [review]

**Review by thomas-roos — APPROVED** [review]

**Pull request merged**